flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/resource/ResourceManager.java

```diff
@@ -83,6 +83,60 @@ public ResourceManager(ReadableConfig config, MutableURLClassLoader userClassLoa
         this.userClassLoader = userClassLoader;
     }
 
+    public boolean exists(Path filePath) throws IOException {
+        boolean exist;
+        FileSystem fs = FileSystem.getUnguardedFileSystem(filePath.toUri());
+        exist = fs.exists(filePath);
+        return exist;
+    }
+
+    /**
+     * register the filePath of flink filesystem. If it is remote filesystem and the file exists
```

Comment:
Nit: JavaDoc comments should start with the capitalized sentence. You can take https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
as a ref.